packages/react-components/react-table/src/components/TableCellLayout/TableCellLayout.types.ts

```diff
@@ -34,4 +40,5 @@ export type TableCellLayoutProps = ComponentProps<Partial<TableCellLayoutSlots>>
 /**
  * State used in rendering TableCellLayout
  */
-export type TableCellLayoutState = ComponentState<TableCellLayoutSlots> & Pick<TableCellLayoutProps, 'appearance'>;
+export type TableCellLayoutState = ComponentState<TableCellLayoutSlots> &
+  Pick<TableCellLayoutProps, 'appearance'> & { size: TableContextValue['size']; avatarSize: AvatarSizes | undefined };
```

Comment:
(nit) You don't seem to be using the new `size` state variable anywhere (unless I'm missing it). I suppose it doesn't hurt to have it, but is it needed?

Reply:
it's used [here](https://github.com/ling1726/fluentui/blob/a847489f765627a35e2ceb3e3066b7a52c4c63ab/packages/react-components/react-table/src/components/TableCellLayout/useTableCellLayoutContextValues.ts#L9) to initialize the context value

Reply:
done in https://github.com/microsoft/fluentui/pull/24807/commits/810164ed937fcc3cb59a565c3c22419079715ce4